scripts/index.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable object-shorthand */
+/* eslint-disable no-undef */
+const form = document.getElementById('add-book');
+let removeBtns;
+
+function addBook(title, author, bookId) {
+  books.push({
+    title: title,
+    author: author,
+    id: bookId,
+  });
+  displayBooks();
+  assignRemoveEventListener();
+}
+
+form.onsubmit = () => {
+  const title = form.elements.title.value;
+  const author = form.elements.author.value;
+  if (books.length) {
+    const bookId = books[books.length - 1].id + 1;
+    addBook(title, author, bookId);
+  } else addBook(title, author, 1);
+  localStorage.setItem('books', JSON.stringify(books));
+  removeBtns = Array.from(booksContainer.querySelectorAll('button'));
+};
+
+function removeBook(removeBtn) {
+  books = books.filter((book) => book.id !== parseInt(removeBtn.id, 10));
+  localStorage.setItem('books', JSON.stringify(books));
+  displayBooks();
+  assignRemoveEventListener();
+}
+
```

Comment:
- [ ] It seems that there is a problem with your remove function because for now, it is removing a book only once, for example, if I add two books and I need to delete them both it does not accept, it makes it possible to delete a book only once. Kindly let's fix this issue 😉

Reply:
I am not sure what you mean but, If you mean you want to delete two books with a certain button that wasn't part of the requirement it was only required to add functionality to remove button that removes correct book

Reply:
He doesn't do that on my end, It's weird @barackm 
He even on my partners

Reply:
https://www.loom.com/share/33d2df9d243547d5b612f1aedeb63e78 
Here's a video of me showing you it works

Reply:
This issue is well resolved! ✔️